src/Microsoft.Dism.Tests/GetTargetEditionsTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c). All rights reserved.
+//
+// Licensed under the MIT license.
+
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Dism.Tests
+{
+    public class GetTargetEditionsTest : DismTestBase
+    {
+        public GetTargetEditionsTest(TestWimTemplate template, ITestOutputHelper testOutput)
+            : base(template, testOutput)
+        {
+        }
+
+        [Fact]
+        public void GetTargetEditionsOnlineSession()
```

Comment:
This test failed on the hosted agent, not sure what edition those machines are using

```
Shouldly.ShouldAssertException : editionCollection
should not be empty but was
```

https://jeffkl.visualstudio.com/Public/_build/results?buildId=1714&view=ms.vss-test-web.build-test-results-tab&runId=1006520&resultId=100073&paneView=debug

On my machine it passes, not sure what to do for this one.  I'd be fine with omitting it but maybe we can find a way to make it work?

Reply:
I think we can keep the test, but just remove the empty check. That way, at least a test ensures the method doesn't fail.